app/helpers/application_helper.rb

```diff
@@ -22,4 +22,10 @@ def devise_mapping
     @devise_mapping ||= Devise.mappings[:user]
   end
 
+  # IE11 reports HTML1500 warnings on the console if tags are not explicitly
+  # closed (like happens if you used `tag`).  Here we're rendering tags with
+  # attributes and no content to be able to parse the JSON in JS.
+  def json_div(options = nil)
```

Comment:
I think we should bias towards shipping, but @alexsoble if you happen to know a better way to do this in Rails that's awesome too.  You have much knowledge of the Rails-y ways :)